README.md

```diff
@@ -57,11 +57,6 @@ Use run_on_single_node parameter when you initialize a workflow (or a Slurm reso
 * Note you cannot use this option with the **exclude_list** option.
 * Note you cannot specify more than one node using this option.
 
-## What are valid root directories for the workflow?
-
-Currently the root directory should be be in your home directory and not on an NAS like `/nas/gaia/` as the submission will fail for an NFS reason.
-The experiment directory can be (and ought to be) on such a drive, though.
-
```

Comment:
Are there any restrictions on valid root dirs?  Should the README include a statement that root dirs can now be on shared /nas/gaia or /nas/user/xyz ?

Reply:
There are no longer any restrictions on valid root dirs. There used to be a NFS related bug when trying to mount a shared NAS as the root dir for Pegasus but it has been resolved.

Reply:
@danielnapierski I've added a note to the readme that the `root_dir` can be set to any location on the NAS, and that a shared drive e.g. `/nas/gaia` may be preferred due to space limitations on the user home dirs. However if unspecified the script will default to a users `/nas/home/xyz/` dir.